app/helpers/dashboard_helper.rb

```diff
@@ -15,6 +15,8 @@ def dashboard_actions
 
   def render_widget(widget)
     render(:partial => widget.template, :locals => widget.data)
+  rescue ActionView::MissingTemplate
+    _("Missing template '%{template}' for widget '%{widget}'. The widget or a plugin providing it was probably removed") % { :widget => widget.name, :template => widget.template }
```

Comment:
Seems reasonable.

Nitpick: add a full stop to the end of the second sentence, since there's one on the first.